argo-apps/tooling-app-of-apps/values.yaml

```diff
@@ -26,5 +26,7 @@ applications:
      path: camelk-community  
    - name: tekton
      path: tekton  
-#    - name: devspaces
-#      path: devspaces  
+   - name: devspaces
```

Comment:
Does this mean we have 2 separate application which deals with Dev Spaces? Is this needed? Can't we have a single app?